apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/page/content/subsection/SubSectionCreator.java

```diff
@@ -37,6 +44,41 @@ public CreateSubSectionResponse createSubSection(long pageId, Long userId, Creat
         }
     }
 
+
+    public DeleteSubSectionResponse deleteSubSection(DeleteSubSectionRequest request) {
+        try {
+            log.info("Deleting sub section");
+            Integer orderNbr = waUiMetaDataRepository.getOrderNumber(request.subSectionId());
+            Long pageNumber = waUiMetaDataRepository.findPageNumber(request.subSectionId());
+            Long nbsComponentUid = waUiMetaDataRepository.findNextNbsUiComponentUid(orderNbr+1, pageNumber);
+            if(nbsComponentUid == 1010L || nbsComponentUid == 1015L || nbsComponentUid == null || nbsComponentUid == 1016L) {
```

Comment:
Similar to above, can we define some constants for these Ids to improve readability?